index.html

```diff
@@ -975,6 +976,67 @@ <h2>Methods</h2>
             selected candidate pair =].
           </p>
         </dd>
+        <dt>
+          <dfn>removeCandidatePairs</dfn>
+        </dt>
+        <dd>
+          <p>
+            The {{removeCandidatePairs}} method immediately removes the provided candidate pairs, making them unavailable to use
```

Comment:
I would avoid words like "immediately" since the ICE agent lives on a different thread, unless "instruct the ICE agent" necessarily needs to be implemented synchronously. But my understanding is that "instruct" could be implemented as "post a task to instruct"... but see my other comment, it is basically the same thing

Reply:
Ack, removed "immediately" to avoid any confusion.